### PR TITLE
New package: Flakron.ApacheMaven version 3.8.9

### PR DESCRIPTION
**Pull request by flakron**
The installer for the package, is built in this repo: https://github.com/flakron/winget-apache-maven

Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? [Issue] (https://github.com/microsoft/winget-pkgs/issues/65391)

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change? - There is one, but it is stale.
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/163496)

### CODE REVIEW

**Comment by flakron** [comment]
@microsoft-github-policy-service agree [company="Flakron Bytyqi"]

**Comment by flakron** [comment]
@microsoft-github-policy-service agree

File: manifests/f/Flakron/ApacheMaven/3.8.9/Flakron.ApacheMaven.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Flakron.ApacheMaven
+PackageVersion: 3.8.9
+InstallerType: zip
+NestedInstallerType: inno
+NestedInstallerFiles:
+- RelativeFilePath: setup.exe
+Installers:
```
Comment (stephengillie) [inline manifests/f/Flakron/ApacheMaven/3.8.9/Flakron.ApacheMaven.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Oracle.JavaRuntimeEnvironment
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @flakron,

- This appears to be the same package as [another PR](https://github.com/microsoft/winget-pkgs/pull/123118) - should the same `PackageIdentifier` be used? 
- This package appears to be a Java application. 
image
 - This means both that:
   - A `JRE` is a dependency
   - Unfortunately we're not currently accepting scripted (non-compiled) applications.

File: manifests/f/Flakron/ApacheMaven/3.8.9/Flakron.ApacheMaven.installer.yaml (hunk 1 above)
Comment (flakron) [reply manifests/f/Flakron/ApacheMaven/3.8.9/Flakron.ApacheMaven.installer.yaml:10]: Unfortunately it depends on the user what Java SDK they use, it is impossible to know what they need to use and force them to the Oracle one.

**Comment by flakron** [comment]
Hi @stephengillie 

- I didn't see where any specific rule in regards to naming the `PackageIdentifier`, But to the point, since Maven is maintained by the Apache Foundation, and I'm not affiliated with them it didn't seem to me right to pretend that the package is built by them, and it seemed a bit disingenuous. 
- The package is a dependency manager for java development. It is dependent on `jdk` not `jre`, `JAVA_HOME` is referred to the jdk install directory (whatever jdk the user is using).
- Unfortunately they will never have a executable binary. So maybe I close the pull request altogether, if the guidelines do not accept such cases (unfortunately since a lot of Java tools will never have binary executables.

**Pull request closed without merging**